angio-frontend/src/modules/analyse/views/AnalyseDetails.vue

```diff
@@ -43,18 +87,24 @@
         public created() {
             this.fetching = true;
             AnalyseApiService.getAnalyseById(this.analyseId)
-                .then((res) => {
-                    this.analyse = res.data.data;
+                .then((analyseRes) => {
+                    AnalyseApiService.getAnalyseStarred(this.analyseId)
+                        .then((starredRes) => {
+                            this.analyse = analyseRes.data.data;
+                            this.analyse.starred = starredRes.data.data.starred;
+                            this.starred = starredRes.data.data.starred;
 
-                    PatientApiService.getPatientById(this.analyse.additionalInfo.patientId)
-                        .then((res) => {
-                            this.patient = res.data.data;
-                            this.setDocumentTitle();
+                            PatientApiService.getPatientById(this.analyse.additionalInfo.patientId)
```

Comment:
Why `PatientApiService.getPatientById(...)` call depends on `AnalyseApiService.getAnalyseStarred(...)`?

Reply:
Fixed

Reply:
`AnalyseApiService.getAnalyseStarred(...)` and `PatientApiService.getPatientById(...)` depends on `AnalyseApiService.getAnalyseById(...)` only. Not from each other.